internal/cloudapi/openapi.yml

```diff
@@ -177,10 +177,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Repository'
-        upload_requests:
-          type: array
-          items:
-            $ref: '#/components/schemas/UploadRequest'
+        upload_request:
+          $ref: '#/components/schemas/UploadRequest'
```

Comment:
Isn't it required to have an upload request?

Reply:
It's marked as required I tihnk, see L168

Reply:
ah right, I didn't expand the view ...